src/styles/blocks/star1.css

```diff
@@ -0,0 +1,3 @@
+.stars--1 :first-child {
```

Comment:
You don't neeed to create a separate file for all modifiers, it should be done in one css file